src/EventStore.Core/Services/Transport/Http/Controllers/InfoController.cs

```diff
@@ -22,6 +26,9 @@ public class InfoController : IHttpController, IHandle<SystemMessage.StateChange
         private readonly IOptions _options;
         private readonly ProjectionType _projectionType;
         private VNodeState _currentState;
+        private List<ScavengeStatus> _scavengeStatusMessages = new List<ScavengeStatus>();
```

Comment:
Interesting design question here: is it useful to maintain a history of these (and if so should they be persisted in a stream?) - or do we actually only care about the status _right now_?